docs/support_scripts/create_json_responses/app.py

```diff
@@ -12,8 +12,8 @@
 
 env_path = current_path / "../../../migration_steps/.env"
 load_dotenv(dotenv_path=env_path)
-base_url = os.environ.get("SIRIUS_FRONT_URL")
-password = os.environ.get("API_TEST_PASSWORD")
+base_url = "http://localhost:8080"
```

Comment:
Reason i'm hard coding these is that we never actually use this script from a container. It's just a helper for creating the api tests and is only ever used locally.